oauth-overview.html.md.erb

```diff
@@ -0,0 +1,106 @@
+---
+title: OAuth Overview 
+owner: API
+---
+
+<strong><%= modified_date %></strong>
+
+## <a id='overview'></a> What is Oauth?
+
+OAuth 2.0 is an open standard for authorization and it provides client applications a 'secure delegated access' to server resources on behalf of a resource owner. OAuth essentially allows access tokens to be issued to third-party clients by an authorization server, with the approval of the resource owner, or end-user. The client uses the access token to access our protected APIs.
+More information you can read at http://oauth.net/2/.
+
+If you are already familiar with OAuth, then all you really need to know about are the two authentication endpoints: 
+
+* authorization endpoint:  https://consent.swisscom.com/c/oauth2/oauth
+* token request endpoint: https://consent.swisscom.com/o/oauth2/token
+
+Before you begin to use an application, you will need a client id, redirect URI, and a client secret key. These details will be used to authenticate your application and verify that the API calls being made are valid.
+
+There are 3 ways to obtain an access token, which can be used instead of a client_id/API key, namely:
+
+* [Authorization Code (3-legged) Tutorial](./oauth-authorisation-code.html)
```

Comment:
Please use the American English "authorization" (instead of "authorisation") for the file name